assistant_dialog_skill_analysis/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.0.2'
+__version__ = "1.0.2"
```

Comment:
Version need to change to 1.1.0

Reply:
this is from 1 of the commits, the final version is updated